python/paddle/incubate/multiprocessing/reductions.py

```diff
@@ -161,11 +169,31 @@ def _reduce_lodtensor(lodtensor):
             if dim == 0:
                 # Empty tensors have nothing be mapped.
                 return (_rebuild_lodtensor_empty, (type(lodtensor),))
+        inmeta = False
+        if os.environ.get('FLAGS_use_file_descripor', True) in [
```

Comment:
Change it be `core.globals()["FLAGS_use_file_descripor"]`. The `core` is `paddle.base.core`.